addons/advanced_ballistics/functions/fnc_initializeTerrainExtension.sqf

```diff
@@ -22,6 +22,7 @@ _initStartTime = CBA_missionTime;
 _mapSize = getNumber (configFile >> "CfgWorlds" >> worldName >> "MapSize");
 
 if (("ace_advanced_ballistics" callExtension format["init:%1:%2", worldName, _mapSize]) == "Terrain already initialized") exitWith {
+    ACE_LOGINFO_1("Terrain already initialized [world: %1]", worldName);
```

Comment:
Terrain loading is persistent between missions.
So if you load it once, it won't have to load again.